lib/LocalVerifiableCredentialStore.js

```diff
@@ -0,0 +1,137 @@
+/*!
+ * Copyright (c) 2019-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+/**
+ * Each instance of this API is associated with a single edv and performs
+ * initialization (ensures required indexes are created).
+ */
+import PouchDB from 'pouchdb';
+import pouchFind from 'pouchdb-find';
+PouchDB.plugin(pouchFind);
+
+export default class LocalVerifiableCredentialStore {
+  constructor({db, dbName, profileId}) {
+    this.db = db || new PouchDB(dbName);
+    this.profileId = profileId;
+    this.db.createIndex({
+      index: {
+        fields: [
+          'content.id',
+          'meta.parentId',
+          'meta.profileId',
+          'meta.displayable'
+        ]
+      }
+    });
+  }
+
+  /**
+   * Gets a verifiable credential by its ID.
+   *
+   * @param {string} id
+   */
```

Comment:
jsdocs throughout are not proper.  Needs jsdoc linting enabled and fix-up.  This one is also missing `@returns`